doc/source/pypi-release-guide.rst

```diff
@@ -33,15 +33,15 @@ Initiate a release process with GitHub issue
 Start pre-release
 ~~~~~~~~~~~~~~~~~
 
-.. _release-instructions-project-owner:
+.. _release-instructions-project-maintainer:
 
 #. Review the release GitHub issue created in the previous step.
 
 #. Set ``PYPI_TOKEN`` and ``PAT_TOKEN`` are configured at the organization or repository level by following the instructions in Appendix :ref:`1 <pypi-token-setup>`, :ref:`2 <pat-token-setup>`, respectively.
 
 #. Setup GitHub pages at the repository level by following the instructions in Appendix :ref:`3 <gh-pages-setup>`.
 
-#. Confirm the ``github_admin_username`` section in ``.github/workflows/build-wheel-release-upload.yml`` is that of the project owner.
+#. Confirm the ``github_admin_username`` section in ``.github/workflows/build-wheel-release-upload.yml`` is that of the project maintainer.
```

Comment:
maintainer for release

Reply:
yes, but to avoid confusion, do we want to change `github_admin_username` to `github_maintainer_username` to avoid confusion?

Reply:
yes, https://github.com/Billingegroup/release-scripts/issues/150 issue created.